fiftyone/migrations/revisions/v1_4_0.py

```diff
@@ -0,0 +1,64 @@
+"""
+FiftyOne v1.4.0 revision.
+
+In previous versions we used the run_link field in delegated operations to
+store the path to the log file. This revision migrates the data from the 
+run_link field to the new log_path field allowing for the use of both fields 
+and a cleaner implementation.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+
+def up(db, dataset_name):
+    _migrate_field_bulk(db, dataset_name, "run_link", "log_path")
+
+
+def down(db, dataset_name):
+    _migrate_field_bulk(db, dataset_name, "log_path", "run_link")
+
+
+def _migrate_field_bulk(db, dataset_name, source_field, target_field):
+    pipeline = [
+        {"$match": {"name": dataset_name}},
+        {
```

Comment:
Any reason to not use `update_many()` directly on the `delegated_ops` collection?

`$lookup + $unwind + $merge` could be significantly slower, but this would only matter if there are many DOs per dataset (there could be, or could not be; hard to say)